admin_manual/office/index.rst

```diff
@@ -0,0 +1,25 @@
+===============
+Office
+===============
+
+.. image:: ../images/office.png
+
+Nextcloud Office supports editing your documents in real time with multiple other editors, showing high fidelity, WYSIWYG rendering and preserving the layout and formatting of your documents.
+
+Users can insert and reply to comments and invite others without a Nextcloud account for anonymous editing of files with a public link shared folder.
+
+Nextcloud Office supports dozens of document formats including DOC, DOCX, PPT, PPTX, XLS, XLSX + ODF, Import/View Visio, Publisher and many more...
+
+Nextcloud Office is based on the Collabora Online Development Edition (CODE) and is available free and under heavy development, adding features and improvements all the time! Enterprise users have access to the more stable, scalable Collabora Online Enterprise based version through a Nextcloud support subscription.
```

Comment:
Looks good a link to [nextcloud.com/enterprise](https://nextcloud.com/enterprise/), what do you think?